include/nuttx/power/battery_ioctl.h

```diff
@@ -144,6 +144,7 @@ enum batio_operate_e
   BATIO_OPRTN_RESET,
   BATIO_OPRTN_WDOG,
   BATIO_OPRTN_SHIPMODE,
+  BATIO_OPRTN_CUTOFF_CURRENT,
```

Comment:
Isn't BATIO_OPRTN_CUTOFF_CURRENT equivalent to BATIO_OPRTN_SYSOFF or even BATIO_OPRTN_HIZ ? What is the purpose in this case?